pages/myforms/[[...path]].tsx

```diff
@@ -126,7 +128,21 @@ const RenderMyForms: NextPageWithLayout<MyFormsProps> = ({ templates }: MyFormsP
 };
 
 RenderMyForms.getLayout = (page: ReactElement) => {
-  return <Template page={page}></Template>;
+  return (
+    <>
```

Comment:
From what I understand, getLayout was used to maintain state across pages (React component tree...). It looks like a Meta Title is missing. Though when I test the MyForms page in different states I see a title.  I'm probably missing something and will just leave this comment here just encase

Reply:
Hmmm let me look this over before merging in